instrumentation/netty-reactor-0.6.0/build.gradle

```diff
@@ -1,21 +0,0 @@
-dependencies {
-    implementation(project(":agent-bridge"))
-    implementation("io.projectreactor.ipc:reactor-netty:0.6.0.RELEASE")
-}
-
-jar {
-    manifest { attributes 'Implementation-Title': 'com.newrelic.instrumentation.netty-reactor-0.6.0' }
-}
-
-verifyInstrumentation {
-    // Capped for the compatibility docs plugin
-    passesOnly 'io.projectreactor.ipc:reactor-netty:[0.6.0.RELEASE,0.7.15.RELEASE]'
```

Comment:
Why was the `netty-reactor-0.6.0` module deleted instead of just being capped at `0.7.0.RELEASE` where the new module comes in? Do we just not support `0.6.n.n` versions any more?

Reply:
I felt like the decision to support 6.0.0 in a way that wasn't equal to 7+ would cause confusion. It's unlikely there are many users of it since it's middleware used by projects like Spring